devops/main.tf

```diff
@@ -0,0 +1,15 @@
+terraform {
+  required_providers {
+    awstest = {
+      source = "harshicorp/aws"
+      version = "~> 5.0"
+    }
+  }
+}
+
+provider "awstest" {
+  region = var.region_name
+  shared_config_files = ["%USERPROFILE%\\.aws\\config"]
```

Comment:
No need to hardcode credentials.
Terraform should pick them up from env vars, you can export which sso profile to use for AWS before running.